docs/benchmark.md

```diff
@@ -0,0 +1,98 @@
+# Benchmarks
+
+## Running the benchmark
+
+You can run the benchmark by executing `./gradlew :android-benchmark:benchmark:connectedCheck`. (Note that the benchmark should be run on an actual device)
+You can then take the output from the benchmark and run it through `DataParser.kt` to get a structured breakdown as seen below.
+
+## Results
+
+Running the benchmark on a Pixel 3 yields the following results:
```

Comment:
Let's link to a spec sheet for the pixel 3 here maybe?

Reply:
bf05483